### ISSUE TITLE
Adjust heading level in "ask a question" and "suggest a correction"

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1763]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Collection page, e.g. https://findingaids.princeton.edu/catalog/C0178

### Description of error / warning

Skipped heading level.

I'm a little confused because wave only reports skipped level for one of these but not the other. But they're both h5 which I don't think makes sense.

### WCAG success criterion

[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)

### WCAG conformance level

A

### Recommended fix

I'm not totally sure. either:
- they should be h1 because the page behind them is disabled
- they should be h2 because they're below the component title, which is h1, in terms of information hierarchy
- they should be h3 because the buttons that activate them are in the tab panel area, and each tab panel has an h2 heading.

### Tool used to find the problem

wave

### Screenshot

<img width="717" height="442" alt="Image" src="https://github.com/user-attachments/assets/eb07d5f5-bd26-4969-ad9d-70cd34a9627e" />
